libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -377,14 +428,23 @@ void AC_PosControl::set_alt_target_from_climb_rate_ff(float climb_rate_cms, floa
     // jerk_z is calculated to reach full acceleration in 1000ms.
     float jerk_z = accel_z_cms * POSCONTROL_JERK_RATIO;
 
-    float accel_z_max = MIN(accel_z_cms, safe_sqrt(2.0f * fabsf(_vel_desired.z - climb_rate_cms) * jerk_z));
+    float accel_z_max = MIN(accel_z_cms, safe_sqrt(2.0f * fabsf(climb_rate_cms - _vel_desired.z) * jerk_z));
```

Comment:
I guess it's used in two places with a third being slightly different because it's missing the fabsf().

Because of the fabsf the order of the args doesn't matter I think so I'll leave it to @lthall to say why he wanted to switch the order.  Maybe it was to make the order consistent in the couple of places it's used.

I've added the const.